modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -3,87 +3,91 @@
-Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
-Install-Module -Force OpenSSHUtils
-Set-Service -Name ssh-agent -StartupType 'Automatic'
-Set-Service -Name sshd -StartupType 'Automatic'
-Start-Service ssh-agent
-Start-Service sshd
-$pubKeyConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PubkeyAuthentication yes','PubkeyAuthentication yes'
-$pubKeyConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$passwordConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PasswordAuthentication yes','PasswordAuthentication yes'
-$passwordConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$authFileConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace 'AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys','#AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys'
-$authFileConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$pubKeyLocationConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace 'Match Group administrators','#Match Group administrators'
-$pubKeyLocationConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-Restart-Service sshd
-New-item -Path $env:USERPROFILE -Name .ssh -ItemType Directory -force
-----
+[IMPORTANT]
+====
+The Windows Server 2004 VM must include the link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]. This patch is required to set the VXLAN UDP port, which is required for clusters installed on vSphere. This patch is not available for the `Windows Server 1809` VM image.
+====
 
 . Install and configure VMware Tools version 11.0.6 or greater on the Windows VM. See the link:https://docs.vmware.com/en/VMware-Tools/index.html[VMware Tools documentation] for more information.
 
 . After installing VMware Tools on the Windows VM, verify the following:
+
 .. The `C:\ProgramData\VMware\VMware Tools\tools.conf` file has the following entry:
 +
 [source,ini]
 ----
 exclude-nics=
 ----
 +
-This entry ensures the following:
+This entry ensures the cloned vNIC generated on the Windows VM by the hybrid-overlay is not ignored.
+
+.. The Windows VM has a valid IP address in vCenter:
++
+[source,posh]
+----
+C:\> ipconfig
+----
+
+.. The VMTools Windows service is running:
++
+[source,posh]
+----
+C:\> Get-Service -Name VMTools | Select Status, StartType
+----
+
+. Install and configure the OpenSSH Server on the Windows VM. See Microsoft's documentation on link:https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse[installing OpenSSH] for more details.
+
+. Set up SSH access for an administrative user. See Microsoft's documentation on the link:https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_keymanagement#administrative-user[Administrative user] to do this. The public key used in the instructions must correspond to the private key you created in the WMCO namespace that holds your secret.
```

Comment:
Have mixed feeling about removing the [Powershell script to configure SSH on vSphere Windows VMs](https://github.com/openshift/openshift-docs/pull/35728/files#diff-a9972807dbbd53e5423f681de126b7ec7a6e26d87bea6a6232b1e8d4e192a686L25).

Everything can be done following [Microsoft documentation](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse), is it worth to provide a helper script in the documentation to automate that process?

cc @openshift/openshift-team-windows-containers

Reply:
I suggest we move away from providing scripts which implies it is something we will update and maintain.

Reply:
Agreed; we really want to stay away from scripts in the official docs if we can. I added the script previously because I wasn't clear how to do it manually. With there being a clear path now, it'll take some of the burden off the team.